app/forms/collection_form.rb

```diff
@@ -3,34 +3,51 @@
 
 # The form for collection creation and editing
 class CollectionForm < Reform::Form
+  extend T::Sig
+
   property :name
   property :description
   property :contact_email
-  property :managers, prepopulator: lambda { |_options|
-    self.managers = model.creator.sunetid
-  }
+  property :managers, default: -> { model.creator.sunetid }
```

Comment:
i stumbled on what seemed like the need for this change when i copypasta'ed the `prepopulator` pattern  for the reviewers field, and noticed that it was only ever presenting the default value, even if i'd saved an updated value to the DB.